cypress/e2e/po/components/cluster-badge.po.ts

```diff
@@ -0,0 +1,53 @@
+import ComponentPo from '@/cypress/e2e/po/components/component.po';
+import LabeledInputPo from '@/cypress/e2e/po/components/labeled-input.po';
+import CheckboxInputPo from '@/cypress/e2e/po/components/checkbox-input.po';
+import ColorInputPo from '@/cypress/e2e/po/components/color-input.po';
+import AsyncButtonPo from '@/cypress/e2e/po/components/async-button.po';
+
+export default class ClusterBadgePo extends ComponentPo {
+  constructor(selector = '.dashboard-root') {
```

Comment:
@yonasberhe23 does this has to be set at the top page level or would it be better to reduce the lookup scope to an inner level locator?

Reply:
good call. updated in https://github.com/rancher/dashboard/pull/9816/commits/6c163ee775d4b9f3efb55152b5825923ebe8d965